index.js

```diff
@@ -169,14 +169,35 @@ exports.login = async function (options) {
 };
 
 exports.viewAccount = async function (options) {
-    let near = await connect(options);
-    let account = await near.account(options.accountId);
-    let state = await account.state();
-    if (state && state.amount) {
-        state['formattedAmount'] = utils.format.formatNearAmount(state.amount);
+    try {
+        let near = await connect(options);
+        let account = await near.account(options.accountId);
+        let state = await account.state();
+        if (state && state.amount) {
+            state['formattedAmount'] = utils.format.formatNearAmount(state.amount);
+        }
+        console.log(`Account ${options.accountId}`);
+        console.log(inspectResponse.formatResponse(state));
+    } catch(e) {
```

Comment:
best to move this error handling into https://github.com/near/near-cli/blob/master/utils/exit-on-error.js so  that  it applies to all commands

Reply:
You are absolutely right that it would be better, but I can see potential problem here.
If I transfer the logic to this file and the user, for example runs the "sendMoney" command (or any other - where there is no accountId, etc.), and the server returns a weird error (for example, Currently error is "[-32000] Server error: ... "with huge stack trace.) then we get a weird wrong error case. I don't know witch cases can potentially cause "[-32000] Server error:"
How would you suggest manage those cases?

Reply:
As far as I know the error for account being not found is:

```
TypedError: [-32000] Server error: account bazinga does not exist while viewing
```

i.e. you can check for "does not exist while viewing" substring.